plugin.php

```diff
@@ -1,7 +1,9 @@
 <?php
 $id = 'xoct';
-$version = '4.0.1';
+$version = '4.0.2';
+$version_check = '44ac530093a998b525b0a73ba536e64f03bbaff47446cf99e1a31d6a042a4549';
```

Comment:
@Amstutz this is just a random string, the same is stored in the the database. on future updates we can check against this (as in [src/Util/UpdateCheck.php](https://github.com/opencast-ilias/OpenCast/pull/74/files#diff-d672a39b1c328ba19e298256dd594f0146838221c300e43c92d5d82ba193297d)